user/common-tasks/disposablevm.md

```diff
@@ -153,6 +153,15 @@ $ qvm-open-in-vm @dispvm:online-dvm-template https://www.qubes-os.org
 
 This will create a new DisposableVM based on `online-dvm-template`, open the default web browser in that DisposableVM, and navigate to `https://www.qubes-os.org`.
 
+#### Example of RPC policies to allow this behavior
+
+In dom0, add the following line to `/etc/qubes-rpc/policy/qubes.OpenURL`
+~~~
+$anyvm @dispvm:online-dvm-template allow
+~~~
```

Comment:
Could you add some words explaining what this is meant to do?

Reply:
Added a basic description